oauth/client_test.go

```diff
@@ -0,0 +1,267 @@
+			PhysicalProduct:    stripe.Bool(false),
+			ProductDescription: stripe.String("product-description"),
+			State:              stripe.String("NV"),
+			StreetAddress:      stripe.String("123 main"),
+			URL:                stripe.String("http://example.com"),
+			Zip:                stripe.String("12345"),
+		},
+	})
+
+	assert.Contains(t, url, "response_type=test-code")
+	assert.Contains(t, url, "stripe_user[block_kana]=block-kana")
+	assert.Contains(t, url, "stripe_user[block_kanji]=block-kanji")
+	assert.Contains(t, url, "stripe_user[building_kana]=building-kana")
+	assert.Contains(t, url, "stripe_user[building_kanji]=building-kanji")
+	assert.Contains(t, url, "stripe_user[business_name]=b-name")
+	assert.Contains(t, url, "stripe_user[business_type]=llc")
+	assert.Contains(t, url, "stripe_user[city]=Elko")
+	assert.Contains(t, url, "stripe_user[country]=US")
+	assert.Contains(t, url, "stripe_user[currency]=USD")
+	assert.Contains(t, url, "stripe_user[dob_day]=15")
+	assert.Contains(t, url, "stripe_user[dob_month]=10")
+	assert.Contains(t, url, "stripe_user[dob_year]=2019")
+	assert.Contains(t, url, "stripe_user[email]=test%40example.com")
+	assert.Contains(t, url, "stripe_user[first_name]=first-name")
+	assert.Contains(t, url, "stripe_user[first_name_kana]=first-name-kana")
+	assert.Contains(t, url, "stripe_user[first_name_kanji]=first-name-kanji")
+	assert.Contains(t, url, "stripe_user[gender]=female")
+	assert.Contains(t, url, "stripe_user[last_name]=last-name")
+	assert.Contains(t, url, "stripe_user[last_name_kana]=last-name-kana")
+	assert.Contains(t, url, "stripe_user[last_name_kanji]=last-name-kanji")
+	assert.Contains(t, url, "stripe_user[phone_number]=999-999-9999")
+	assert.Contains(t, url, "stripe_user[physical_product]=false")
+	assert.Contains(t, url, "stripe_user[product_description]=product-description")
+	assert.Contains(t, url, "stripe_user[state]=NV")
+	assert.Contains(t, url, "stripe_user[street_address]=123+main")
+	assert.Contains(t, url, "stripe_user[url]=http%3A%2F%2Fexample.com")
+	assert.Contains(t, url, "stripe_user[zip]=12345")
+}
+
+// RoundTripFunc.
+type RoundTripFunc func(req *http.Request) *http.Response
+
+// RoundTrip.
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
+
+// NewTestClient returns *http.Client with Transport replaced to avoid making
+// real calls
+func NewTestClient(fn RoundTripFunc) *http.Client {
+	return &http.Client{
+		Transport: RoundTripFunc(fn),
+	}
+}
+
+func StubConnectBackend(httpClient *http.Client) {
+	mockBackend := stripe.GetBackendWithConfig(
+		stripe.ConnectBackend,
+		&stripe.BackendConfig{
+			URL:        "https://localhost:12113",
+			HTTPClient: httpClient,
+		},
+	)
+	stripe.SetBackend(stripe.ConnectBackend, mockBackend)
+}
+
+func TestNewOAuthToken(t *testing.T) {
+	stripe.Key = "sk_123"
+
+	// stripe-mock doesn't support connect URL's so this stubs out the server.
```

Comment:
Maybe consider changing all instances of "URL's" to "URLs"? I Googled and there's [no clear answer for the correct way to handle these](https://english.stackexchange.com/a/921) (sources vary), but my feeling is that the apostrophe-less version is generally preferred.

Reply:
Updated these instances.